keyboards/handwired/dactyl_manuform/5x6_right_trackball/pmw3360.h

```diff
@@ -19,6 +19,9 @@
 #pragma once
 
 #include "spi_master.h"
+#include "wait.h"
+#include "debug.h"
+#include "print.h"
```

Comment:
These should go in the .c files, as there's nothing here that needs them. The console ifdef below can be removed too.